src/VolleyManagement.Backend/VolleyManagement.Domain/Dto/UserInRoleDto.cs

```diff
@@ -25,9 +25,11 @@ public UserInRoleDto()
         /// </summary>
         public string UserName { get; set; }
 
+#pragma warning disable S3956 // see more https://stackoverflow.com/questions/25528649/exception-on-inner-linq-query-when-calling-tolist?rq=1
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not suppress issues. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1309)